### ISSUE TITLE
The image size is 640x640 or shorter side is 1200, long side is less than 1600?

### ISSUE THREAD

**Issue by hdjsjyl** [deepinsight/mxnet-SSH#10]
I used 640x640 to get trained model, it only can get mAP of 68.8 on hard subset of validation set of WIDER FACE. In the original paper, the image is that shorter side is 1200, and long side is not longer than 1600. So can you give me some advice? Thanks

**Comment by nttstar** [comment]
Training with 640, but testing with any size you want.

**Comment by hdjsjyl** [comment]
@nttstar ， I think the parameters are the same as what you said. But the results I got are far from the results you got. It is very strange. Any advice will be appreciated. Thanks

**Comment by hdjsjyl** [comment]
@nttstar  what is the batch size, 8 or 16?  Thanks

**Comment by nttstar** [comment]
use default setting in config.py
